drivers/pinctrl/pinctrl_gecko_s1.c

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
nb: this seems to duplicate a lot of code with `pinctrl_gecko.c`, couldn't a single driver be created?

Reply:
I've merged the two drivers and used `CONFIG_SOC_GECKO_SERIES1` symbol to make sure that proper parts are compiled. Could you take a look at the current solution?